skins/oasis/css/core/ads.scss

```diff
@@ -584,3 +574,50 @@ body > a > img[src$="noad.gif"] {
 #WikiaArticleMsg {
 	display: none;
 }
+
+/** Floating ads **/
+
+/*** Incontent ***/
+#WikiaArticle {
+	&.floating {
+		z-index: initial !important;
+
+		.wikia-ad {
+			bottom: $wikia-bar-wrapper-height + $floating-ad-margin;
+			height: $video-container-width !important;
```

Comment:
`!important`s burn my eyes :fire: Have you tried other solutions, so we can get rid of them?